jcapi.go

```diff
@@ -240,26 +278,31 @@ func (jc JCAPI) extractStringArray(input []interface{}) []string {
 	return returnVal
 }
 
-func getStringOrNil(input interface{}) string {
-	returnVal := ""
-
+func getStringOrNil(input interface{}) (s string) {
 	switch input.(type) {
 	case string:
-		returnVal = input.(string)
+		s = input.(string)
 	}
 
-	return returnVal
+	return
 }
 
-func getUint16OrNil(input interface{}) uint16 {
-	var returnVal uint16
-
+func getUint16OrNil(input interface{}) (i uint16) {
 	switch input.(type) {
 	case uint16:
-		returnVal = input.(uint16)
+		i = input.(uint16)
 	}
 
-	return returnVal
+	return
+}
+
+func getIntOrNil(input interface{}) (i int) {
+	switch input.(type) {
+	case int:
+		i = input.(int)
+	}
+
+	return
 }
```

Comment:
True, but that function is going to disappear as soon as I can eliminate the rest of the manual marshal/unmarshal.